src/components/Input/Input.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+import style from "./Input.module.css";
+
+type inputProps = {
+  value: string;
+  placeholder: string;
+  onChange: (event: React.FormEvent) => void;
+};
+
+const Input: React.FC<inputProps> = ({
+  value,
+  placeholder,
+  onChange,
+}: inputProps) => {
+  return (
+    <input
+      className={style.search__input}
+      type="text"
+      placeholder={placeholder}
+      value={value}
+      onChange={onChange}
+    ></input>
```

Comment:
Можно сократить запись, потому что дочерних элементов нет
```suggestion
    />
```